src/Terminal.ts

```diff
@@ -37,14 +37,17 @@ export default class Terminal extends events.EventEmitter {
 
     createInvocation(): void {
         var invocation = new Invocation(this.currentDirectory, this.dimensions, this.history);
-        invocation.once('end', () => {
-            if (app.dock) {
-                app.dock.bounce('informational');
-            }
-            this.createInvocation();
-        }).once('working-directory-changed', (newWorkingDirectory: string) =>
-            this.setCurrentDirectory(newWorkingDirectory)
-        );
+
+        invocation
+            .once('clear', _ => this.clearInvocations())
```

Comment:
I understand, but there  is no command typing part of the story in this PR, right? And shouldn't be for that matter, because it's not about Windows support.